cli/tests/badly_formatted_fixed.md

```diff
@@ -0,0 +1,15 @@
+<!-- Not using an extension to avoid triggering dprint. -->
```

Comment:
What do you mean by this comment? You can exclude files from `dprint` formatting in `.dprintrc.json`

Reply:
I guess I followed `badly_formatted.mjs`. I updated it.